apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import { ProgressBar } from '@fluentui/react-progress';
+import { PauseAnimationDecorator, TestWrapperDecoratorFullWidth } from '../utilities/TestWrapperDecorator';
 
 storiesOf('ProgressBar converged', module)
+  .addDecorator(PauseAnimationDecorator)
+  .addDecorator(TestWrapperDecoratorFullWidth)
```

Comment:
A few things:
* It'd probably be better to add use a fixed width wrapper, not full width, so the screenshots aren't so wide.
* The order of TestWrapperDecoratorFixedWidth and PauseAnimationDecorator needs to be flipped, to have the pause animation decorator on the outside. 
* You'll also need to add a StoryWright decorator to crop to the testWrapper like @sopranopillow mentioned.

```suggestion
  .addDecorator(TestWrapperDecoratorFixedWidth)
  .addDecorator(PauseAnimationDecorator)
  .addDecorator(story => (
    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
  ))
```